scss/_footer.scss

```diff
@@ -0,0 +1,168 @@
+.footer .navbar-nav {
+  --#{$prefix}nav-link-font-weight: #{$font-weight-bold};
+
+  display: flex;
```

Comment:
Don't have a strong opinion on this, but wouldn't it be better to use the utility classes for `display: flex`, `flex-direction`, etc. in order to reduce the bundle size?

Reply:
IMO, if this is a mandatory class for the component or if the components is used with a particular attribute, it should be inside the CSS file otherwise, utilities could be used. IMO in here, there's nothing that follows these cases.